### PR TITLE
Feat(Tour): Extended Tour to Hud

### PR DESCRIPTION
**Pull request by rkenyon1969**
#174 

@wski 
@akonsowski 
@blynch11p

### CODE REVIEW

File: app/js/tour/hudTour.js [hunk 1]
```diff
@@ -0,0 +1,246 @@
...
+var contentLocalCenter = '<button class="btn btn-sm btn-default" data-role="next"  onclick="parent.location.href=\'' + CENTER_URL + '\'">Next Center &raquo;</button>';
+var contentLocalStart = '';
+var contentLocalTemplate = '';
+
+if(tourDB.library === true){
+  contentLocalBookmarks = "When you bookmark a listing in Center, it appears here in your HUD. Bookmarks provide easy access to listings. Use them to group and access your tools. Click a tile to quickly launch the bookmark.";
+  contentLocalTemplate = '<div class="popover" role="tooltip" tabIndex="0"> <div class="arrow"></div> <h3 class="popover-title"></h3> <div class="popover-content"></div> <div class="popover-navigation" style="width:300px;"> <button class="btn btn-sm" id="end-tour-btn" data-role="end" tabIndex="0">End tour</button> <div class="btn-group"> <button class="btn btn-sm btn-default" data-role="prev" tabIndex="0">&laquo; Prev</button> <button class="btn btn-sm btn-default" data-role="next" tabIndex="0">Next &raquo;</button> <button class="btn btn-sm btn-default" data-role="pause-resume" data-pause-text="Pause" data-resume-text="Resume" tabIndex="0">Pause</button> </div> </div> </div>';
```
Comment (blynch11p) [inline app/js/tour/hudTour.js]: Just a general comment... 
For readability, I recommend breaking up long strings (particularly HTML in string format) and formatting it how you would expect to see it in an HTML file.  It's easier to track down layout issues and make small tweaks quickly.

Example:
```
contentLocalTemplate = '<div class="popover" role="tooltip" tabIndex="0">'
    + '<div class="arrow"></div>'
    + '<h3 class="popover-title"></h3>'
    + '<div class="popover-content"></div>'
    + '<div class="popover-navigation" style="width:300px;">'
        + '<button class="btn btn-sm" id="end-tour-btn" data-role="end" tabIndex="0">End tour</button>'
        + '<div class="btn-group">'
            + '<button class="btn btn-sm btn-default" data-role="prev" tabIndex="0">&laquo; Prev</button>'
            + '<button class="btn btn-sm btn-default" data-role="next" tabIndex="0">Next &raquo;</button>'
            + '<button class="btn btn-sm btn-default" data-role="pause-resume" data-pause-text="Pause" data-resume-text="Resume" tabIndex="0">Pause</button>'
        + '</div>'
    + '</div>'
    + '</div>';
```

**Comment by blynch11p** [comment]
:+1:

**Comment by blynch11p** [comment]
:+1:

**Comment by akonsowski** [comment]
:-1: does not work  i am getting this error when trying to open hud 
bootstrap-tour.js:531 Uncaught TypeError: $element.popover is not a function

**Comment by wski** [comment]
:+1: I don't know why it's not working on the CentOS systems. I suspect network issues we can discuss internally.

**Pull request merged**